app/models/miq_policy.rb

```diff
@@ -36,6 +36,8 @@ class MiqPolicy < ApplicationRecord
 
   @@built_in_policies = nil
 
+  UI_FOLDERS = [Host, Vm, ContainerImage].freeze
```

Comment:
This line is tough for me ... it has both "UI" and "Container" in a generic model class.

Leave it like this for now (I'm sure worse examples can be found in some legacy code).  But, it would be nice to figure out how to extract these details so that we don't have "UI" things in a model and "Container" things in a generic class.